modules/recommended-etcd-practices.adoc

```diff
@@ -19,3 +19,38 @@ only accepts key reads and deletes. Some of the key metrics to monitor are
 database usage after a history compaction, and
 `etcd_debugging_mvcc_db_total_size_in_bytes` which shows the database size
 including free space waiting for defragmentation.
+
+Etcd writes data to disk, so its performance strongly depends on disk performance. Etcd 
+persists proposals on disk. Slow disks and disk activity from other processes might cause long 
+fsync latencies, causing etcd to miss heartbeats, inability to commit new proposals to the disk 
+on time, which can cause request timeouts and temporary leader loss. It is highly recommended to 
+run etcd on machines backed by SSD/NVMe disks with low latency and high throughput.
+
+Some of the key metrics to monitor on a deployed {product-title} cluster 
+are p99 of etcd disk write ahead log duration and the number of etcd leader changes.
+Use Prometheus to track these metrics. `etcd_disk_wal_fsync_duration_seconds_bucket`
+reports the etcd disk fsync duration, `etcd_server_leader_changes_seen_total` reports 
+the leader changes. To rule out a slow disk and confirm that the disk is reasonably fast, 
+99th percentile of the `etcd_disk_wal_fsync_duration_seconds_bucket` should be less than 10ms. 
+
+Fio, a I/O benchmarking tool can be used to validate the hardware for etcd before or after 
+creating the OpenShift cluster. Run fio and analyze the results:
+
+Assuming container runtimes like Podman or Docker are installed on the machine under test and 
```

Comment:
Podman and Docker are capitalized here, but lowercased later.

Reply:
We should consistently use lower case.

Reply:
Fixed it, thanks.